app/services/cocina/prettier.rb

```diff
@@ -3,18 +3,6 @@
 module Cocina
   # Pretties a cocina object
   class Prettier
-    def self.clean(...)
-      new(...).clean
-    end
-
-    def self.pretty(...)
-      new(...).pretty
-    end
-
-    def self.json(...)
-      new(...).json
-    end
```

Comment:
Dead code.

Reply:
Dead code, but by convention we have static methods for most of our services. If I were to go to use this service in the future, I'd be surprised that it didn't have static methods.

(I don't feel too strongly about this.)

Reply:
I don't feel *very* strongly, but maybe a bit more strongly than you in this case 😆 Why? My sense is that we generally prefer a just-in-time approach to a just-in-case one, lest we accumulate dead, untested, YAGNI code. I'm in favor of dropping these until we need them.